hpcbench/benchmark/osu.py

```diff
@@ -285,7 +287,27 @@ def categories(self):
     def arguments(self):
         """Dictionary providing the list of arguments for every
         benchmark"""
-        return self.attributes['arguments']
+        if 'arguments' in self.attributes:
+            print(
+                "WARNING: 'arguments' use in OSU yaml configuration file is deprecated. Please use 'options'!"
+            )
+            if isinstance(self.attributes['arguments'], dict):
+                return self.attributes['arguments']
+            else:
+                d = {k: self.attributes['arguments'] for k in self.categories}
+                return d
+        elif 'options' in self.attributes:
+            if isinstance(self.attributes['options'], dict):
+                return self.attributes['options']
+            else:
```

Comment:
`self.attributes['options']` is accessed several times, maybe it is worth to put it in a local variable

Reply:
Done!